build.gradle

```diff
@@ -9,6 +9,7 @@ plugins {
 	id 'com.diffplug.spotless' version '6.20.0'
 	id 'com.bmuschko.docker-spring-boot-application' version '9.3.2'
 	id "com.google.protobuf" version "0.9.4"
+	id "io.freefair.lombok" version "8.3"
```

Comment:
very nice :rocket: 
https://docs.freefair.io/gradle-plugins/8.3/reference/#_lombok_plugins

Reply:
thanks for support!!!